packages/terra-clinical-result-view/src/IconCritical.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```

Comment:
Should these be in terra-icon?

Reply:
terra-icon has these icons with pre-filled colors. These ones are to make icons theme-able.